tests/Functional/EventListener/SwitchUserListenerTest.php

```diff
@@ -61,47 +57,63 @@ public function testInvalidateContext()
         $mock = \Mockery::mock(Varnish::class);
         $mock->shouldReceive('invalidateTags')
             ->once()
-            ->with(['fos_http_cache_hashlookup-test']);
+            ->with(\Mockery::any())
```

Comment:
i liked that this verifies that the correct tag is invalidated.

Reply:
Yeah I know, I wasn't to happy about this change. The thing because we do a real login the session gets the standard random id which we can't predict. I don't know mockery well enough but a pattern match is something we can do here so we do check part of the string?

This didn't for me 🤔  `->with(\Mockery::pattern('/^fos_http_cache_hashlookup\-.*/'))`